src/flatc.cpp

```diff
@@ -641,6 +626,10 @@ FlatCOptions FlatCompiler::ParseFromCommandLineArguments(int argc,
       } else {
         // Look up if the command line argument refers to a code generator.
         auto code_generator_it = code_generators_.find(arg);
+        if(code_generator_it == code_generators_.end()) {
+          Error("unknown commandline argument: " + arg, true);
+        }
+
         if (code_generator_it != code_generators_.end()) {
```

Comment:
This line is redundant with the check above. Just throw a `return` in the above condition to reduce nesting here.

Reply:
Done